deploy/setup_cloudflared.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+#-----------------------------------------------------------------------------------------------------------#
+# This script sets up a deployment environment on an Ubuntu 20.04/22.04
+#
+# Documentation:
+# https://developers.cloudflare.com/cloudflare-one/connections/connect-networks/get-started/create-local-tunnel/
+#
+#-----------------------------------------------------------------------------------------------------------#
+
+
+# Add Cloudflare’s package signing key
```

Comment:
If we want to integrate these into our setup pipeline we could wrap them to make them idempotent:

```bash
# Add Cloudflare’s package signing key
if [ -f /usr/share/keyrings/cloudflare-main.gpg ]
then
  echo "Cloudflare gpg key installed at /usr/share/keyrings/cloudflare-main.gpg"
else
  sudo mkdir -p --mode=0755 /usr/share/keyrings
  curl -fsSL https://pkg.cloudflare.com/cloudflare-main.gpg | sudo tee /usr/share/keyrings/cloudflare-main.gpg >/dev/null
fi
```

Reply:
The command is already idempotent.